napari/layers/_tests/test_serialize.py

```diff
@@ -27,6 +28,16 @@ def test_attrs_arrays(Layer, data, ndim):
     # excluding affine transform and `cache` which is not yet in `_get_state`
     assert len(properties) == len(signature.parameters) - 2
 
+    # remove deprecated properties. We do it here cause we want them to still exist and match
```

Comment:
This can be removed now right?

Reply:
tested locally and you're right so have removed